content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,120 @@
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end and can be set on each stack. However, by creating an environment with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+### Repository
+
+Configure the repository and optional subdirectory to use when committing your new project code.
+
+{{% notes "info" %}}
+If you granted the Pulumi GitHub app access to _all_ repositories, the New Project Wizard will allow users to create projects with Deployments enabled in new repositories.
+
+If the app only has access to _some_ repositories, users will only be able to create new projects with Deployments enabled in _existing_ repositories.
+{{% /notes %}}
+
+The GitHub owner is not configurable, as that must match the Pulumi GitHub app's owner in order to work with Deployments.
+
+### Deployment settings
+
+Configure the behavior of Deployments, including when to trigger new Deployments and any environment variables required for the Pulumi program to run.
+
+A full description of each setting is available [here](/docs/pulumi-cloud/deployments/reference/#deployment-settings).
+
+### Deployment method
+
+Finally, select Pulumi Deployments as the deployment method and **Create project** to trigger a deployment. You will see your new deployment start within your **Stacks Deployment** tab.
+
+![Animation of how to create a new project in Pulumi](./pulumi-new-project-wizard.gif)
+
+When your deployment is complete, you can now manage and deploy your infrastructure from the Pulumi Cloud Console. In your stack's **Actions** tab, use the **[Click to Deploy](https://www.pulumi.com/docs/pulumi-cloud/deployments/reference/#click-to-deploy)** button to trigger a deployment.
+
+This feature is useful for day-to-day operational tasks, such as debugging a stuck stack or rectifying drift by performing a refresh. With Click to Deploy, you can execute these actions without needing to pull the stack and its source code onto your local machine.
+
+![Image of the Pulumi cloud console actions](./pulumi-cloud-actions.png)
+
+## Cleanup
+
+To avoid incurring any unwanted charges, clean up the resources you created:
+
+1. Navigate to your stack in the Pulumi console.
+2. Delete the stack, by clicking on the **Settings** tab and the **Delete stack** button.
+3. Delete the GitHub repository you created.
+
+## Next steps
+
+In this tutorial, you set up a basic push-to-deploy stack using Pulumi’s new project wizard. You configured your stack to use the GitHub app, deployed resources, and explored the fundamental Pulumi Deployment browser workflow.
```

Comment:
A natural next step would also be to push a change to your newly provisioned GitHub repo and see the deployment get triggered! And/or submit a pull request and see a PR preview.

Reply:
Yup totally and that's the next tutorial which i'll be linking to.